src/components/datepicker/calendar.scss

```diff
@@ -79,9 +81,16 @@ md-calendar {
 
 // Contains the scrolling element (this is the md-virtual-repeat-container).
 .md-calendar-scroll-container {
+  // Add an inset shadow to help cue users that the calendar is scrollable. Use a negative x
+  // offset to push the vertical edge shadow off to the right so that it's cut off by the edge
+  // of the calendar container.
+  box-shadow: inset -3px 3px $md-calendar-scroll-cue-shadow-radius rgba(black, 0.2);
```

Comment:
All of the whiteframes were more complicated than what I wanted to do here, which is add a shadow to *only* the top of element (and to be `inset` on top of that). The whiteframes, as I understand them, are for raising an entire box to a certain level.